scss/main.scss

```diff
@@ -765,6 +769,11 @@ $animation-length: 80s;
     z-index: 1;
     margin: 0px 50px;
 }
+
```

Comment:
can you also add `transition: .3s ease;` to the `sponsor-img` class to smooth out the expand effect

Reply:
Done! Way smoother!